### PR TITLE
Update CRDS dependency

### PR DESCRIPTION
**Pull request by stscieisenhamer**
**Description**

Update the CRDS dependency to the Build 7.9 versions of CRDS

Checklist
- [ ] Tests
- [ ] ~Documentation~
- [ ] ~Change log~
- [x] Milestone
- [x] Label(s)

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -31,7 +31,7 @@ install_requires =
     asdf>=2.8.2
     astropy>=5.0.0
     BayesicFitting>=2.7.2
-    crds>=11.5.2
+    crds~=11.6.0
```
Comment (hbushouse) [inline setup.cfg:34]: For my own education, what kind of comparison does the "~" do? Is that like any 11.6.x subversion?

**Review by hbushouse — APPROVED** [review]
Looks OK to me.

File: setup.cfg (hunk 1 above)
Comment (hbushouse) [reply setup.cfg:34]: So I guess you could use ">=11.6.0" to get any 11.6.x subversion, but that would also allow future 11.7, 12.0, etc. releases to satisfy the condition. This way you're still restricting it to 11.6.x only. Interesting.

Comment (stscieisenhamer) [reply setup.cfg:34]: Feel more comfortable with this, at least for Build releases. development and master can be reverted to a looser restriction.

**Pull request merged**